doc/level0.rst

```diff
@@ -212,24 +223,25 @@ characteristics, and it could alert the client if it discovers one.
 If the client finds an Autocrypt Setup Message, it should offer to
 import it to enable Autocrypt.  If the user agrees to do so:
 
- * The client should prompt the user for their corresponding Setup
-   Code as a UTF-8 string.  If there is an ``Autocrypt-Setup-Message:
-   v1`` OpenPGP header, the client MAY present the user with a
+ * The client prompts the user for their corresponding Setup
+   Code.  If the ``Autocrypt-Setup-Message: v0`` header contains the parameter
+   ``input-type=alphanumeric``, then the client MAY choose to present the user with a
    specialized input dialog that better assists the user with input in
-   the particular format described above.  If no
-   ``Autocrypt-Setup-Message: v1`` OpenPGP header is present, the
-   client MUST provide a plain UTF-8 string text entry.
+   the particular format described above.
+   If there is no ``input-type`` attribute, or the ``input-type`` is unknown,
+   then the client MUST provide a plain UTF-8 string text entry.
+
  * The client should try decrypting the message with the supplied
    Setup Code.  If it decrypts:
+
  * The client should verify that the User ID on the key matches the
    User ID on the relevant account.
+
  * If it does, the client should import the secret key material and
    announce to the user that the import was successful.
 
-.. todo::
-
-   should we delete the message after import?  if the user declines to
-   import it, should we delete it?
+The message SHOULD be kept after importing it because it can serve
+as a recovery mechanism for the secret key.
```

Comment:
My reason is that i don't want to leave the secret key in a place where it's easy (or standard) to find, because that could be combined with the Setup Code (which we've asked the user explicitly to write down) in order to decrypt all previously encrypted messages.